app/Services/Contact/Contact/UpdateContactFoodPreferences.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Services\Contact\Contact;
+
+use App\Services\BaseService;
+use App\Models\Contact\Contact;
+use Illuminate\Validation\ValidationException;
+
+class UpdateContactFoodPreferences extends BaseService
+{
+    /**
+     * Get the validation rules that apply to the service.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'account_id' => 'required|integer|exists:accounts,id',
+            'contact_id' => 'required|integer|exists:contacts,id',
+            'food_preferences' => 'nullable|string',
```

Comment:
Would it be necessary to limit the length of the string, like 255 chars max for instance?

Reply:
Ok the column is a longtext, which is - I guess - 65535 max.